### ISSUE TITLE
startTime for Spans gets set to past value when using parTupled

### ISSUE THREAD

**Issue by LiuVII** [ccadllc/cedi-dtrace#75]
Maybe somehow related to the previous issue when notes and ids were misconfigured
When recording traces noticed that once the parallel operation is used on spans `startTime` gets set to some past value.
Thought that maybe `timeUnit` gets misconfigured but doesn't seem to be an issue.
Here's some output of Trace (`traceId`, `startTime`, `timeUnit`)
```
(08437b2120d84563aca69abb21cb11cc.cb2e8f22e93e9acd<:8304c1927248e81b,989495477311,MICROSECONDS)
(08437b2120d84563aca69abb21cb11cc.d6eb183e2d6020c6<:8304c1927248e81b,989495477485,MICROSECONDS)
(08437b2120d84563aca69abb21cb11cc.8304c1927248e81b<:7b32416c35daf7d9,1556858256813000,MICROSECONDS)
(08437b2120d84563aca69abb21cb11cc.7b32416c35daf7d9<:27e11df15d90203a,1556858256813000,MICROSECONDS)
(08437b2120d84563aca69abb21cb11cc.27e11df15d90203a<:47ead56bdca393c5,1556858256813000,MICROSECONDS)
```
First two correspond to `1970-01-12 10:51:35 +0000` (these were called with `parTupled`) while other three `2019-05-03 04:37:36 +0000`

**Comment by sbuzzard** [comment]
oh, weird - I'll take a look - thanks!

**Comment by sbuzzard** [comment]
You are correct - the unit isn't misconfigured but it's misused during translate. blah.  I guess I'll be releasing 2.0.1.  I really need better tests for parallel.   :-)

https://github.com/ccadllc/cedi-dtrace/blob/master/core/shared/src/main/scala/com/ccadllc/cedi/dtrace/TraceSystem.scala#L148

**Comment by sbuzzard** [comment]
Released 2.0.1 with fix for this.

**Comment by LiuVII** [comment]
seems to be working properly, closing this
